lib/server.js

```diff
@@ -42,7 +42,7 @@ function sendResource(reply, resource) {
  *
  * @type {FastifyPluginCallback<PluginOptions>}
  */
-export default function (fastify, opts, done) {
+export function Server(fastify, opts, done) {
```

Comment:
Nit: it feels a little weird for this to be capitalized because it's not a constructor. Would `createServer` be better?

Reply:
yeah probably. was just matching the name of the export from the index file to reduce the number of changes needed 😄

Reply:
should I update the export from the index file too, or just this file?

Reply:
opted to update the index file too. just break all of the things